apps/server/src/external/demo/app.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * This is a very minimal example for a websocket client
+ * You could use this as a starting point to creating your own interfaces
+ *
+ * This example does not handle disconnections
```

Comment:
if you want to handle reconnect, you can check how the code is done in Ontime, we dont use any libraries here.
There also seems to be plenty of code in stack overflow about this, let me know if you want help